README.md

```diff
@@ -193,46 +201,55 @@ and [create a simulation job](https://docs.aws.amazon.com/robomaker/latest/dg/cr
 
 Procedurally generate an occupancy map for any gazebo world. This map can then be plugged in to navigate a robot in Worldforge worlds. For other aws-robotics worlds, this procedure is optional for the use-cases mentioned in this README. 
 
-### Pre-build
+### Script for default worlds/config
 
-```bash
-# Install dependencies (Ubuntu >18.04)
-sudo apt-get install ruby-dev libxml-xpath-perl libxml2-utils
-```
+Currently the following worlds are supported:
+- [`bookstore`](https://github.com/aws-robotics/aws-robomaker-bookstore-world)  
+- [`small_house`](https://github.com/aws-robotics/aws-robomaker-small-house-world)  
+- [`small_warehouse`](https://github.com/aws-robotics/aws-robomaker-small-warehouse-world)  
+- [`no_roof_small_warehouse`](https://github.com/aws-robotics/aws-robomaker-small-warehouse-world)
+- worldforge
 
+
+To generate a map, simply run
 ```bash
-# Fetch and install ROS dependencies
-cd simulation_ws
-rosws update
-rosdep install --from-paths src -r -y
-cd ..
+./scripts/genmap_script.sh <world_name>
 ```
+where `<world_name>` can be any value in the list above. Extra steps below are needed for `worldforge`.
+
+
+#### Generating map for WorldForge worlds
+
+After exporting a world from WorldForge, we can unzip the content and move under simulation_ws package:
 
-For Worldforge worlds,
 ```bash
-# Extract and move Worldforge exported worlds
 unzip exported_world.zip
 mv aws_robomaker_worldforge_pkgs simulation_ws/src/
+
+#For worldforge worlds, set WORLD_ID to the name of your WF exported world (eg: generation_40r67s111n9x_world_3),
+export WORLD_ID=<worldforge-world-name>
+
+# Run map generation script
+./scripts/genmap_script.sh worldforge
 ```
 
-### Generate Occupancy Map via map generation plugin
+### Custom worlds/config
```

Comment:
Can be renamed to: Generate map for custom worlds with custom cofig

Reply:
Fixed